lucetc/src/decls.rs

```diff
@@ -125,12 +145,16 @@ impl<'a> ModuleDecls<'a> {
                     let funcid = clif_module
                         .declare_function(&export_symbol, Linkage::Export, signature)
                         .context(LucetcErrorKind::TranslatingModule)?;
+                    exports.push(ExportFunction {
+                        fn_idx: LucetFunctionIndex::from_u32(function_names.len() as u32),
+                        names: exportable_sigix.export_names.clone(),
```

Comment:
All the export names for a function get bunched together, which makes this really straightforward!

Reply:
I think the way the imports and exports collection live in different branches of this name calculation means we can't support an export of an imported function. Can you move those out to the higher level (check for them at the same indentation level as `let name =`, if that makes sense), and also add a test case somewhere that shows we can run an export of an imported function?